docs/get-started/drive-rover.md

```diff
@@ -26,7 +29,17 @@ date: "2024-07-31"
 cost: "0"
 ---
 
-Follow this guide to write code that makes a rover drive in a square.
+If you haven't read through [Learn Viam](/get-started/), we recommend you do so, as it will provide you with an overview of the Viam platform.
+
+The next few quickstarts, will guide you through the concepts you need to get started with Viam.
```

Comment:
```suggestion
These quickstarts will guide you through the concepts you need to get started with Viam.
```
since they're already on one of the quickstarts, and there aren't any more except for these

Reply:
```suggestion
This is the first of a few quickstarts which will guide you through the concepts you need to know to get started with Viam.
```